exercises/sum-of-multiples/canonical-data.json

```diff
@@ -118,6 +118,24 @@
         "limit": 10000
       },
       "expected": 0
+    },
+    {
+      "description": "the only multiple of 0 is 0",
+      "property": "sum",
+      "input": {
+        "factors": [0],
+        "limit": 1
+      },
+      "expected": 0
+    },
+    {
+      "description": "multiples of 5 primes up to 100",
```

Comment:
Working on fixing the diff; seems I had a fork of this repo hanging around I didn't remember ever making. Sigh. Will fix commit message as well.

Personally agree with the lack of the need for `up to 100` but I put it in there to remain consistent with all the other descriptions; the limit isn't really the important part of any of those cases either.

Reply:
My *personal* preference is to favour making sure that new additions focus on what's important, and a preference for this over consistency with other cases. It is true that it is a problem for other cases, but I would only suggest adding to the problem if the benefit to doing so overcomes the addition. I was thinking that the benefit of the consistency did not outweigh.

Reply:
This is interesting because I agree that this case is most necessary for solutions using the inclusion-exclusion principle; solutions using filtering that get to this point likely can already handle this.

I don't have a strong opinion on whether the test description needs to be so specific as to the implementations that it targets. I'll let you make that choice and accept either outcome, though I suspect it will be useful if the description still mentions cardinality > 3